client/extensions/woocommerce/app/stats/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+export default class Stats extends Component {
+
+	render() {
+		return (
+			<Main className="stats woocommerce__main">
```

Comment:
Instead of `woocommerce__main`, use `woocommerce` here. We are name-spacing our CSS under it.